src/app/components/Pagination/Pagination.jsx

```diff
@@ -27,10 +27,11 @@ class Pagination extends React.Component {
     const intPage = parseInt(page, 10);
     const pageNum = type === 'Next' ? intPage + 1 : intPage - 1;
     const svg = type === 'Next' ? <RightArrowIcon /> : <LeftArrowIcon />;
+    const url = this.props.createAPIQuery({ page: pageNum });
```

Comment:
I tried enter no search keywords and just click search button and it will lead me to this url `http://local.nypl.org:3001/search?q=` or on dev `http://dev-discovery.nypl.org/search?q=`.  Then if I click pagination button, the url will be `http://local.nypl.org:3001/search?q=&page=2`. I don't think it break anything.

I think the reason why is `basicQuery` always returns `return `q=${query}${filterQuery}${sortQuery}${fieldQuery}${pageQuery}`;`